src/functions/formatImpressions.js

```diff
@@ -1,12 +1,11 @@
-export function formatImpressions(input) {
-  // Receives an array of objects as inputs and substitutes empty or null impressions or clicks with 0.
-  for (let row = 0; row < input.length; row++) {
-    if (!input[row].Impressions) {
-      input[row].Impressions = 0;
+export const formatImpressions = input =>
```

Comment:
please rename, it's not just about impressions, right?

Reply:
68a467e19db63ad1b79877352a58518bf74b56ea